docs/content/dagster-cloud/getting-started.mdx

```diff
@@ -1,7 +1,198 @@
+
+If you find that Serverless isn't right for you, you can switch to a hybrid deployment at any time.
+
+**To use this deployment type**, click **Activate** and confirm when prompted. **Note**: It may take a few minutes to complete the setup.
+
+---
+
+## Step 3: Deploy your code
+
+Now that you've set up your deployment, you'll need to tell it the location of your Dagster code.
+
+A code location specifies a single Python package or file that defines your Dagster code. Each Dagster Cloud deployment can include code from one or more code locations. Refer to the [Adding a code location guide](/dagster-cloud/developing-testing/code-locations) for more info.
+
+<details>
+	<summary><strong>USING THE GITHUB APP</strong></summary>
+
+1. Click **Connect to Github** and sign into your Github account.
+2. After Dagster Cloud is connected, the following fields will display:
+   - **Repository name** - Enter the name of the repository
+   - **Repository visibility** - Select the visibility of the repository
+3. When finished, click **Clone and Deploy**.
+
+</details>
+
+<details>
+	<summary><strong>USING THE DAGSTER-CLOUD CLI</strong></summary>
+
+To complete this step using the CLI, you can use your own Dagster code or the [`dagster-cloud-examples` code](https://github.com/dagster-io/dagster-cloud/tree/main/dagster-cloud-examples). Our examples use the `dagster-cloud-examples` package.
+
+To install the `dagster-cloud` CLI and the `dagster-cloud-examples` package, run:
+
+```shell
+pip install dagster-cloud dagster-cloud-examples
+```
+
+Select your agent type below to add a new location:
+
+<details>
+	<summary><strong>LOCAL AGENTS</strong></summary>
+
+1. Install the `dagster-cloud-examples` package in your Python environment:
+
+   ```shell
+   pip install dagster-cloud-examples
+   ```
+
+2. In Dagster Cloud, click **Workspace**.
+
+3. Click the **Add sample code location** button:
+
+   <Image
+   alt="Dagit Workspace Screen"
+   src="/images/dagster-cloud/getting-started/adding_code_location.png"
+   width={1118}
+   height={451}
+   />
+
+4. A pre-filled YAML editor with the following config will display:
+
+   ```yaml
+   location_name: cloud-examples
+   code_source:
+     package_name: dagster_cloud_examples
+   ```
+
+   This config tells the agent to load jobs from the `dagster_cloud_examples` Python package.
+
+5. Click **Add code location** to save the config.
+
+The agent will attempt to load the sample code. After the code is loaded, the new code location will show a **Loaded** status and a sample Dagster job will display in Dagster Cloud.
+
+</details>
+
+<details>
+	<summary><strong>AMAZON ECS, KUBERNETES, OR DOCKER AGENTS</strong></summary>
```

Comment:
serverless too?

Reply:
I'm admittedly a bit fuzzy on this part. As Serverless users don't have direct contact with their agent, what should they be doing? Following the instructions for ECS/the other agents, since it's AWS under the hood?

Reply:
I believe the `workspace` CLI is not recommended for serverless users, but cc @prha